source-code/ide-extension/src/main.ts

```diff
@@ -96,12 +114,15 @@ async function main(args: { context: vscode.ExtensionContext }): Promise<void> {
 	args.context.subscriptions.push(
 		vscode.languages.registerCodeActionsProvider(
 			[
-				// TODO: improve with #348
```

Comment:
@samuelstroschein The TODO was because of the activation event of the extension, not the code action. We also need a smart default here, as simply "every file" (which is also possible) is too broad. Should probably also be set by ide-extension-plugin and could be overridden by the user.